.github/workflows/packcheck.yml

```diff
@@ -0,0 +1,199 @@
+      # Where to find the required tools
+      # ------------------------------------------------------------------------
+      PATH: /opt/ghc/bin:/sbin:/usr/sbin:/bin:/usr/bin
+      #TOOLS_DIR: /opt
+
+      # ------------------------------------------------------------------------
+      # Location of packcheck.sh (the shell script invoked to perform CI tests ).
+      # ------------------------------------------------------------------------
+      # You can either commit the packcheck.sh script at this path in your repo or
+      # you can use it by specifying the PACKCHECK_REPO_URL option below in which
+      # case it will be automatically copied from the packcheck repo to this path
+      # during CI tests. In any case it is finally invoked from this path.
+      PACKCHECK: "./packcheck.sh"
+      # If you have not committed packcheck.sh in your repo at PACKCHECK
+      # then it is automatically pulled from this URL.
+      PACKCHECK_GITHUB_URL: "https://raw.githubusercontent.com/composewell/packcheck"
+      PACKCHECK_GITHUB_COMMIT: "91d76691e5cb82e77564be55d0126906b16d76bc"
+
+      # ------------------------------------------------------------------------
+      # Final build variables
+      # ------------------------------------------------------------------------
+      PACKCHECK_COMMAND: ${{ matrix.command }} ${{ matrix.pack_options }}
+
+    runs-on: ${{ matrix.runner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        # The reason we have name here is to force an additional config instead
+        # of adding to an existing config while adding additional configs.
+        # Look at "jobs.<job_id>.strategy.matrix" in github actions to learn
+        # more about adding matrix elements.
+        # Adding any element to the list will increase the number of matrix
+        # elemets proportional to the cross product.
+        name: [ci]
+        # Uncomment GHC versions you want the CI to run on.
+        ghc_version:
+          [ 8.10.1
+          # , 8.8.3
+          # , 8.6.5
+          # , 8.4.4
+          ]
+        command:
+          [ cabal-v2
+          # , stack
+          ]
+        runner:
+          [ ubuntu-latest
+          # , macos-latest
+          ]
+        include:
+          # Additional ci matrix elements
+          # - name: ci
+          #   ghc_version: 8.8.3
+          #   command: stack
+          #   runner: ubuntu-latest
+
+          # - name: ci
+          #   ghc_version: 8.4.4
+          #   command: stack
+          #   runner: ubuntu-latest
+
+          # - name: ci
+          #   ghc_version: 8.8.3
+          #   command: cabal-v2
+          #   runner: macos-latest
+
+          - name: hlint
+            command: cabal-v2
+            runner: ubuntu-latest
+            pack_options: >-
+              HLINT_OPTIONS="lint"
+              HLINT_TARGETS="src"
+
+          # IMPORTANT: Please change the --repo-token to your repo token
```

Comment:
This doesn't work, there is no proper support for github actions in hpc coveralls

Reply:
I did run all the CI's before commenting them. All of them seem to succeed. When you say it won't work does it mean it gives a false positive?

Reply:
Ping @pranaysashank

Reply:
I'm removing hpc-coveralls for now in github CI.